tests/integration/docusaurus/connecting_to_your_data/database/snowflake_yaml_example.py

```diff
@@ -94,4 +94,10 @@
         "default_inferred_data_connector_name"
     ]
 )
-validator.execution_engine.engine.close()
+
+# NOTE: snowflake connector requires connection and engine to be closed and disposed separately.
```

Comment:
Could you make this a `TODO` (maybe in the `<WILL>` format you have below)? Same question for all notes.

Reply:
done